attributes/default.rb

```diff
@@ -35,9 +35,21 @@
 default['hops']['conf_dir']                    = node['hops']['conf_dir_parent'] + "/hadoop"
 default['hops']['share_dir']                    = node['hops']['base_dir'] + "/share/hadoop"
 
-default['hops']['dn']['data_dir']              = "file://" + node['hops']['data_dir'] + "/hdfs/dn"
-default['hops']['dn']['data_dir_permissions']  = '700'
-default['hops']['nn']['name_dir']              = "file://" + node['hops']['data_dir'] + "/hdfs/nn"
+default['hops']['enable_cloud_storage']        = "false"
+default['hops']['cloud_provider']              = "AWS"
```

Comment:
Can you default this to `install/cloud` attribute? At least when we're doing a full AWS installation we won't need to set it in two places.

Reply:
fxd